app/javascript/app/components/emission-pathways-scenario-table/emission-pathways-scenario-table-selectors.js

```diff
@@ -49,19 +69,29 @@ export const getSelectedCategoryOption = createSelector(
   }
 );
 
-export const filterDataByBlackList = createSelector(
-  [scenarioIndicatorsData],
-  data => {
-    if (!data || isEmpty(data)) return null;
-    const whiteList = remove(
-      Object.keys(data[0]),
-      n => ESP_BLACKLIST.indicators.indexOf(n) === -1
-    );
-    return data.map(d => pick(d, whiteList));
+export const getLocationOptions = createSelector([getLocations], locations => {
+  if (!locations) return null;
+  return locations.map(location => ({
+    value: location.id.toString(),
+    label: location.name
+  }));
+});
+
+export const getSelectedLocationOption = createSelector(
+  [getLocations, getLocationSelected],
+  (locations, selectedLocation) => {
+    if (!locations) return null;
+    const location = locations.find(l => l.id.toString() === selectedLocation);
```

Comment:
Can you not just filter from the getLocationsOptions, so that you dont have a selected value before you have your locations? As you have done with categories before?